### PR TITLE
feat: [PIE-4863]: remove @wings-software alias everywhere

### PR DESCRIPTION
**Pull request by udayvunnam**
### Summary

- It's been a few months since the backward compatibility for @harness -> @wings-software is added expecting individuals to migrate, but this is not happening
- This makes it inconsistent across repo and sometimes dual usage. Making an attempt to replace all in a safe way
- Keeps the aliasing only in build tooling(webpack, swc) for some time for MFEs to transition

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by udayvunnam** [comment]
retrigger typecheck

**Review by kapilkumawat86 — DISMISSED** [review]

**Comment by udayvunnam** [comment]
retrigger build

File: src/framework/Templates/TemplateConfigModal/__tests__/__snapshots__/TemplateConfigModal.test.tsx.snap [hunk 1]
```diff
@@ -479,7 +479,7 @@ exports[`CREATE MODE if form changesupdate the preview card: changed value shoul
             class="StyledProps--font StyledProps--main StyledProps--flex StyledProps--flex-justifyContent-center StyledProps--padding-top-medium"
           >
             <div
-              class="css-21x249"
+              class="css-12eylzo"
```
Comment (abhinavrastogi-harness) [inline src/framework/Templates/TemplateConfigModal/__tests__/__snapshots__/TemplateConfigModal.test.tsx.snap:482]: is this a required change?
obfuscated class names should not be present in tests ideally.

**Review by abhinavrastogi-harness — APPROVED** [review]

File: src/framework/Templates/TemplateConfigModal/__tests__/__snapshots__/TemplateConfigModal.test.tsx.snap (hunk 1 above)
Comment (udayvunnam) [reply src/framework/Templates/TemplateConfigModal/__tests__/__snapshots__/TemplateConfigModal.test.tsx.snap:482]: There was no change in the component that generates this snapshot except for updating imports(harness/uicore and harness/design-system). Yes, such snapshots are bad and hard to reason about.

Comment (udayvunnam) [reply src/framework/Templates/TemplateConfigModal/__tests__/__snapshots__/TemplateConfigModal.test.tsx.snap:482]: Also, verified the UI where these snapshots are updated.
image

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**